designs/administrator/olm/operand-creation-at-operator-install/index.md

```diff
@@ -0,0 +1,36 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Allow Operand (Custom Resource) to be Created During Operator Installation
+
+Currently some operators need the user to instantiate a custom resource for the operator to be fully functional, but it is challenging for the user to determine they need to do so today. This design allows the operator developer to specify a single required custom resource that should be created at the same time as the operator is installed.
+
+## Installing an Operator with a Required Operand Specified
+
+![Install operator](img/2-1-install-needCR.png)
+- The user selects the operator they'd like to install and selected installation options.
+- The required custom resource is marked with a badge as **Required**.
+
+![Installing operator](img/2-2-installing.png)
+- The ‘Installing’ screen appears, though the required custom resource is now shown with the **Create [Custom Resource]** button not yet enabled.
+- The user can navigate away from this screen, but they may be more inclined to stay seeing that the custom resource creation is next.
+
+![Installed create CR](img/2-3-instantiateCR.png)
+- Once the operator has successfully installed, the user is notified that they still need to create the required custom resource to use the operator.
+
+![Create custom resource](img/2-4-createCR.png)
```

Comment:
This may be out of scope (if so, definitely ignore!) but the form says "Select at least 3 nodes..." and the table only shows 2. Is this possible or maybe just a mockup coincidence? If it is possible, is there an action the user can take or will the creation fail?

Reply:
This is just sort of a generic existing screen to convey the 'create CR' part of this design, but I will tweak it to be right.